Sources/tasks/CreatePost-Notify.php

```diff
@@ -194,18 +197,38 @@ public function execute()
 			else
 				continue;
 
+			$receiver_lang = empty($data['lngfile']) || empty($modSettings['userLanguage']) ? $language : $data['lngfile'];
+
+			// Censor and parse BBC in the receiver's language. Only do each language once.
+			if (empty($parsed_message[$receiver_lang]))
+			{
+				if ($receiver_lang != $user_info['language'])
```

Comment:
Err, I don't think that `$user_info` is available in cron.

Reply:
Derp. Although `$user_info` itself is, `$user_info['language']` isn't set.

Reply:
Fixed in 3d64b0e